eea/facetednavigation/widgets/tagscloud/counter.py

```diff
@@ -38,17 +39,17 @@ def query(self, cid, **kwargs):
 
         # Count
         count = getattr(widget, 'count', lambda brains, sequence: {})
-        res = count(brains, sequence=vocabulary.keys())
+        res = count(brains, sequence=list(vocabulary.keys()))
         res.pop("", 0)
         oll = res.pop('all', 0)
 
-        res = res.items()
+        res = list(res.items())
         res.sort(key=operator.itemgetter(1), reverse=True)
 
         maxitems = widget.maxitems
         if maxitems:
             res = res[:maxitems]
-        res.sort(key=operator.itemgetter(0), cmp=compare)
+        res.sort(key=lowercase)
```

Comment:
As mentioned in the original pull-request this is probably wrong since lowercase uses the second element for the comparison, were the original implementation here compared the first.

Reply:
I don't think this is a big issue. Actually, the new behavior may be the desired one :)